products/ubuntu2004/profiles/stig.profile

```diff
@@ -568,6 +568,7 @@ selections:
     - clean_components_post_updating
 
     # UBTU-20-010450 The Ubuntu operating system must use a file integrity tool to verify correct operation of all security functions.
+    - aide_build_database
```

Comment:
could you please put this line after the `package_aide_installed`?

Reply:
Thanks for the feedback, this was originally added in this commit / PR by accident and will need to be refactored: https://github.com/ComplianceAsCode/content/pull/11059

I'll drop the change in the PR listed above, and rebase this PR with the change.

Reply:
thank you for the feedback. Added this and rebased the PR.